### DISCUSSION TITLE
Plan for Studio

### DISCUSSION THREAD

**Discussion by WebPirat** [GrapesJS/studio#2] [General]
Hello,

Do you have any plans for the studio you can share at the moment? Will it go open-source, or will there be an option to pay for a license to self-host/package and rebrand it for our own products? The alpha looks awesome, so I am curious.

Best Regards
WebPIrat

**Comment by artf** [comment]
Yes @WebPirat, I can share something already.

Studio is built as an application but with reusability in mind, so the idea is to allow other applications to embed the Studio editor via its own SDK, built on top of GrapesJS.

The goal of Studio SDK is to allow you to embed a white-label editor in your application easily, GrapesJS API is still there under the hood (eg. you can still use GrapesJS plugins for blocks, components, etc.) but you don't need to learn all of its APIs and you have already a good accessible UI around it. 
GrapesJS core is leaning toward being more headless, with less UI around it.

I'll try to provide a way to try this SDK first, but the full functionality will require a license.
